about.html

```diff
@@ -0,0 +1,159 @@
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My page</a></li>
+                <li><a href="#">Logout</a></li>
+                
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-menu">
+                <a href="/index.html">
+                <img src="images/fashion.png" alt="fashion">
+                <h3>Fashion Show <br> Conference</h3>
+                </a>
+            </div>
+            <ul>
+                <li id="link"><a href="./about.html">About</a></li>
+                <li><a href="#">Program </a></li>
+                <li><a href="#">Sponsor </a></li>
+                <li><a href="#">Join </a></li>
+                <li><a href="#">News</a></li>
+                <li class="campaign"><a href="#"> CC Campaign</a></li>
+            </ul>
+        </nav>
+
+    </header>
+   <main>
+    <section class="topic-about">
+    <div id="menu-bar"></div>
+    <i id="menubar" class="fa fa-bars" aria-hidden="true"></i>
+    <div class="main-header">
+        <h2>"Hello Fashion world"</h2>
+        <h1>FASHION CURATORS <br>
+            GLOBAL MEET 2022
+            </h1>
+            <hr class="hori">
+        <p>The 2022 edition of FS2022 is organised in collaboration with 
+            AMD – Akademie Mode & Design – Hochschule Fresenius – University 
+            of Applied Sciences. The AMD Akademie Mode & Design offers internationally 
+            certified programs in fashion. <span>
+            Promoting and supporting creative as well as business-oriented youth remains an essential
+             strategic focus of the AMD Akademie Mode & Design for over 30 years of its dedicated educational 
+             activities. To enable students to create solutions for future sustainable design and fashion to 
+             become change makers for the creative industry. </span>
+            </p>
+        <div class="details">
+            <span>Kindly contact us through this email, incase you have<br>
+            any futher questions about submit.<br>
+            <u>fashion2022@fashionconference.org</u></span>
+        </div>
+        </div>
+        </section>
+        <section class="about-intro">
+            <h2>Curator Showcase 
+                  Global Fashion motto 2022
+            </h2>
+            <hr class="hori">
+            <p>We are passionate about young curators and would
+                love to make creative ideas recongnized.
+                <span>We invite researchers from academy and industry, and 
+                    professionals from the world of Fashion to take part in <br>
+                    FS2022 reflecting on how to make sustainable fashion 
+                    not just a dream, but indeed a reality we are all engaged
+                     with.</span>
+            </p>
+        </section>
+        <section class="event">
+            <h1>.</h1>
```

Comment:
- [ ] For semantic HTML best practices, please make sure that each HTML document has only one <h1> tag. This also helps with SEO 👍

Reply:
- [ ] @MahmoudBakr23  Thank you for the observation. I have changed it to `<h5>` tag.